contributing/development/core_and_modules/internal_rendering_architecture.rst

```diff
@@ -269,7 +268,7 @@ RenderingDevice presents a similar level of abstraction as Metal or WebGPU.
 
 **Vulkan RenderingDevice implementation:**
 
-- `drivers/vulkan/rendering_device_vulkan.cpp <https://github.com/godotengine/godot/blob/4.0/drivers/vulkan/rendering_device_vulkan.cpp>`__
+- `drivers/vulkan/rendering_device_vulkan.cpp <https://github.com/godotengine/godot/blob/master/drivers/vulkan/rendering_device_vulkan.cpp>`__
```

Comment:
This 404's because `rendering_device_vulkan.cpp` became `rendering_device_driver_vulkan.cpp`